charts/kube-plex/values.yaml

```diff
@@ -44,6 +44,9 @@ claimToken: ""
 # Set the timezone of the plex server
 timezone: Europe/London
 
+# Set extra environment variables
```

Comment:
This could use an example or even a mention that the syntax is the same as with the deployment environments.